index.html

```diff
@@ -8,7 +8,7 @@
   <link rel="stylesheet" href="https://unpkg.com/highlight.js/styles/vs2015.css" id="highlight-theme" />
   <style type="text/css">
     .reveal p {
-      text-align: left;
+      text-align: justify;
```

Comment:
Now bullet points are left aligned and paragraphs justified, it sometimes looks a bit funny...

Reply:
Easy, don't use bullet points ( ͡° ͜ʖ ͡°)